db/seeds/01_web_app.rb

```diff
@@ -1,3 +1,3 @@
 # frozen_string_literal: true
 
-Doorkeeper::Application.create_with(name: 'Web', redirect_uri: Doorkeeper.configuration.native_redirect_uri, scopes: 'read write follow push').find_or_create_by(superapp: true)
+Doorkeeper::Application.webapp
```

Comment:
Two things here ...

- Stylewise, feels sort of weird to rely on the error-catching behavior of what reads like a finder method to create data here ... not the worst thing, just a little odd. Maybe use a data creating bang method in the module and call it from here and also the error path in the module?
-  Does this change the idempotent nature of this line of code? I don't think so, but want to confirm ... previously it would only create a record here if something with `superapp: true` did not exist; otherwise it would find that one and not create another. I think that same behavior might still be enforced at the db level because of the index on superapp, but it's sort of changed at the ruby level, I think...?

Reply:
It doesn't, and we've other cases of this such as the Instance Actor account.